platforms/ios/src/TangramMap/TGHttpHandler.mm

```diff
@@ -74,8 +86,7 @@ - (NSMutableDictionary *)HTTPAdditionalHeaders {
 - (void)setAdditionalHTTPHeaders:(NSMutableDictionary *)HTTPAdditionalHeaders {
     _HTTPAdditionalHeaders = HTTPAdditionalHeaders;
     self.configuration.HTTPAdditionalHeaders = HTTPAdditionalHeaders;
-    //Probably unnecessary, but for safety sake
-    self.session.configuration.HTTPAdditionalHeaders = HTTPAdditionalHeaders;
+    self.session = [NSURLSession sessionWithConfiguration:self.configuration];
```

Comment:
To be consistent, we would want to do this in `setCachePath` as well, right?

Reply:
Agreed. Fixed.